cypress/support/config/allServices.js

```diff
@@ -0,0 +1,1552 @@
+const isLive = appEnv => appEnv === 'live';
+const isTest = appEnv => appEnv === 'test' || appEnv === 'stage';
+const isLocal = appEnv => appEnv === 'local';
+
+const allServices = appEnv => ({
```

Comment:
You'll need to update everywhere we import the services in the Cypress tests if renaming this to allServices

Reply:
The only reference is in `cypress/support/config/services.js` but I will double check all of the Cypress tests are running & passing OK before pushing